metricsaggregator/src/main/java/io/dockstore/metricsaggregator/MetricsAggregatorS3Client.java

```diff
@@ -84,10 +84,15 @@ public void aggregateMetrics(ExtendedGa4GhApi extendedGa4GhApi) {
                 continue; // Continue aggregating metrics for other directories
             }
 
-            getAggregatedMetrics(executions).ifPresent(metrics -> {
-                extendedGa4GhApi.aggregatedMetricsPut(metrics, platform, toolId, versionName);
-                System.out.printf("Aggregated metrics for tool ID %s, version %s, platform %s from S3 directory %s%n", toolId, versionName, platform, directory);
-            });
+            try {
+                getAggregatedMetrics(executions).ifPresent(metrics -> {
+                    extendedGa4GhApi.aggregatedMetricsPut(metrics, platform, toolId, versionName);
```

Comment:
Interesting, it's possible for the tool to no longer exist when putting aggregated metrics. Didn't think of that, thanks!